Description: Increase the default printWidth for examples from 77 to 129
 This decreases the probability of examples escaping the "reproduciblePaths"
 function due to line wrapping.
Author: Doug Torrance <dtorrance@piedmont.edu>
Forwarded: not-needed
Last-Update: 2020-12-03

--- a/M2/Macaulay2/m2/run.m2
+++ b/M2/Macaulay2/m2/run.m2
@@ -111,7 +111,7 @@
      cmd = cmd | readmode(ArgNotify,      "--notify");
      cmd = cmd | readmode(ArgSilent,      "--silent");
      cmd = cmd | readmode(ArgStop,        "--stop");
-     cmd = cmd | readmode(ArgPrintWidth,  "--print-width 77");
+     cmd = cmd | readmode(ArgPrintWidth,  "--print-width 129");
      cmd = cmd | concatenate apply(srcdirs, d -> (" --srcdir ", format d));
      -- TODO: fix capture, add preloaded packages to Macaulay2Doc, then delete the following two lines
      needsline := concatenate(" -e 'needsPackage(\"",pkgname,"\", Reload => true, FileName => \"",pkg#"source file","\")'");
